tileview/src/main/java/com/qozix/tileview/tiles/TileCanvasViewGroup.java

```diff
@@ -261,9 +263,6 @@ void generateTileBitmap( Tile tile ) {
   }
 
   void addTileToCurrentTileCanvasView( Tile tile ) {
-    if( mTilesAlreadyRendered.contains( tile ) ) {
-      return;
-    }
```

Comment:
hm after looking through more of the changes, i think my comment above https://github.com/moagrius/TileView/pull/261/files#r52124981 is probably not relevant.  what's the rationale here?